src/smt/theory_str_noodler/theory_str_noodler.cpp

```diff
@@ -375,12 +375,12 @@ namespace smt::noodler {
 
                 
                 
-                return;
+                //return;
             }
 
             // build axiom 2: Length(a_str) == 0 <=> a_str == ""
             {
-                return;
+                // return;
```

Comment:
I believe we can remove these returns now.

Reply:
Refactoring is on my todo list. But not now